_posts/2020-05-28-on-reading-comprehension-original-thoughts.md

```diff
@@ -0,0 +1,224 @@
+
+#### Limitations of reading the originals
+Of course, there are practical limitations to a *foundational works first* approach. Sometimes they simply aren’t the right source. For example we want to get get an overview. A summary might yield better results here. Another hindrance can be language. We decide to read a book written in a foreign language we haven’t mastered. In that case, we only have two options, we learn the language or we use a translation. Which one is more appropriate depends solely on the reader and his preferences. In the end, the goal is understanding but the time investment most be sensible.
+
+One limitation specific to the digital reader is the availability of the work as digital copy. If it doesn’t exist it cannot be integrated into the reader’s setup. Fortunately, for many foundational works the [Gutenberg Project](https://www.gutenberg.org/) did a tremendous job in providing different digital formats at no cost. We also can find prominent books in its digital form on platforms like Amazon. Where there is a demand, there will be supply.
+
+### Derivatives for getting an overview or extending our knowledge
+Starting with the foundations makes sure we are building up the same foundations in our head, a prerequisite for continued reading and research. But not every art and science is equally interesting or important us. When only an overview or rough grasp of the concepts of a field is required working through a summary can be an advisable approach. 
+
+On the other hand, when we already have a basic understanding we might extend that with new criticism we didn’t find on our own, new ideas and concepts which derive from the originals, or to bring the foundations into a modern context.
+
+### Conclusion
+What type of written work we select will depend on multiply factors and we have to analyse them before we make our decision. But if our goal should be thorough understanding of a field we are well advised to start with the foundational works, because they will provide us with the necessary basis.
+
+## Final words
+I will use the final words of this text to emphasis two points important to any reader who might decide to adopt the approach described above. First, as everything it has its limitations. This process cannot be applied to every book. For example, the content of some written works is truly dense and using highlights would extract significant parts of the text, which doesn’t make for a compressed representation. More on that in [Appendix 3](#3:%20Educational%20and%20dense%20books) . This reading approach worked for many historic foundational books and papers, but it can fail with other types. We have to keep that in mind and continue experimenting and improving this reading approach when friction arises.
```

Comment:
```suggestion
I will use the final words of this text to emphasise two points important to any reader who might decide to adopt the approach described above. First, everything it has its limitations. This process cannot be applied to every book. For example, the content of some written works is truly dense and using highlights would extract significant parts of the text, which doesn’t make for a compressed representation. More on that in [Appendix 3](#3:%20Educational%20and%20dense%20books) . This reading approach worked for many historic foundational books and papers, but it can fail with other types. We have to keep that in mind and continue experimenting and improving this reading approach when friction arises.
```

* starting a clause with *as* needs to be balanced, like *as this, then that*, but since there was no second bit there, I just removed the *as*

Reply:
Works for me, but unfortunately you forgot to remove `everything`: `First, everything it has its limitations`. I guess it should be `First, it has its limitations.`